packages/app/obojobo-document-engine/src/scripts/oboeditor/components/code-editor.js

```diff
@@ -113,6 +114,16 @@ class CodeEditor extends React.Component {
 		})
 	}
 
+	renameModule(label) {
```

Comment:
This code is duplicated in page-editor as well - Want to make sure this happens in the same place in case one of these functions are modified